rpcapd/rpcapd.c

```diff
@@ -91,36 +91,39 @@ static void main_cleanup_childs(int sign);
 */
 static void printusage(void)
 {
```

Comment:
This is a separate change that just cleans up the code and the usage message, and thus applies regardless of the build system or the target platform; please submit it as a separate pull request.